source/botbotbot.py

```diff
@@ -119,6 +119,9 @@ def recv_message(self, content='', parent=None, this_message=None, sender='', se
                 self.pause(pause_text='/me has been paused by "' + sender + '".', reply_to=this_message)
         elif self.paused and EuphUtils.command('!help', self.nickname).match(content):
             self.pause(reply_to=this_message)
+        elif content == '!antighost':
```

Comment:
Please change this line to `elif EuphUtils.command('!antighost').match(content)`. This makes it consistent with the lines around it and more flexible (e.g. matches case-insensitively, strips whitespace, etc.).